docs/developing/path.md

```diff
@@ -0,0 +1,101 @@
+Sandstorm apps live at ephemeral URLs. This page documents the
+details.
+
+## Web URLs
+
+When a Sandstorm grain is running, Sandstorm uses a different URL for
+each session. This means even for a single grain, each different user
+visiting the site will have a different base URL. There isn't a stable
+URL like `https://myapp.me.sandcats.io/` for an app to use.
```

Comment:
Maybe link to the security properties page with a hint as to why our platform behaves this way, since it's somewhat unusual.